### PR TITLE
added query income store

### PR DESCRIPTION
**Pull request by mariyazaiceva**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java [hunk 1]
```diff
@@ -97,4 +101,16 @@ public void deleteByIdAndProfileId(Long id, Long profileId) {
     public void deleteAllByProfileId(Long profileId) {
         storeRepository.deleteAllByProfileId(profileId);
     }
+
+    @Override
+    public PeriodOfIncomeDTO getIncome(long profileId) {
+        Map<PeriodOfIncome, Integer> map = new HashMap<>();
```
Comment (OlgaEvdokimova) [inline src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java:107]: Please, use Map.of(...)

File: src/main/java/internship/mbicycle/storify/dto/PeriodOfIncomeDTO.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PeriodOfIncomeDTO {
```
Comment (OlgaEvdokimova) [inline src/main/java/internship/mbicycle/storify/dto/PeriodOfIncomeDTO.java:15]:
```suggestion
public class IncomePeriodDTO {
```

File: src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java (hunk 1 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java]: Done

File: src/main/java/internship/mbicycle/storify/dto/PeriodOfIncomeDTO.java (hunk 2 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/dto/PeriodOfIncomeDTO.java]: Done

**Review by OlgaEvdokimova — APPROVED** [review]

**Pull request merged**